plugins/notification/src/index.ts

```diff
@@ -169,6 +179,10 @@ export interface DocUpdateTx {
   modifiedBy: Ref<Account>
   modifiedOn: Timestamp
   isNew: boolean
+  title?: IntlString
```

Comment:
Why they are optional? And we discussed rename to InboxItem

Reply:
I made it optional to mitigate issues with a previous existing data and migration. We've postpone the idea of renaming things. We've started from the idea to change Inbox implementation too, but we've decided not continue with it and just add fields to create more beautiful desktop notifications. So no other impact than desktop notification with all of this.

Reply:
I think we always intend to make our code base better, and not hold unattended stuff.